docs/pages/deploy-a-cluster/deployments/aws-starter-cluster-terraform.mdx

```diff
@@ -0,0 +1,714 @@
+at rest](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/EncryptionAtRest.html).
+
+### cluster_name
+
+```code
+export TF_VAR_cluster_name="<Var name="teleport-example" />"
+```
+
+This is the internal Teleport cluster name to use. This should be unique, and not contain spaces, dots (.) or other
+special characters. Some AWS services will not allow you to use dots in a name, so this should not be set to a domain
+name. This will appear in the web UI for your cluster and cannot be changed after creation without rebuilding your
+cluster from scratch, so choose carefully. A good example might be something like `teleport-<company-name>`.
+
+### ami_name
+
+```code
+export TF_VAR_ami_name="gravitational-teleport-ami-ent-(=teleport.version=)"
+```
+
+Gravitational automatically builds and publishes OSS, Enterprise and Enterprise FIPS 140-2 AMIs when we
+release a new version of Teleport. The AMI names follow the format: `gravitational-teleport-ami-<type>-<version>`
+where `<type>` is either `oss` or `ent` (Enterprise) and `version` is the version of Teleport e.g. `(=teleport.version=)`.
+
+FIPS 140-2 compatible AMIs (which deploy Teleport in FIPS 140-2 mode by default) have the `-fips` suffix.
+
+The AWS account ID which publishes these AMIs is `126027368216`. You can list the available AMIs with
+the example `awscli` commands below. The output is in JSON format by default.
+
+<Admonition
+  type="tip"
+  title="List Gravitational AMIs"
+>
+  OSS AMIs<br/>
+  ```code
+  aws --region <Var name="us-west-2" /> ec2 describe-images --owners 126027368216 --filters 'Name=name,Values=gravitational-teleport-ami-oss-(=teleport.version=)'
+  ```
+
+  Enterprise AMIs<br/>
+  ```code
+  aws --region <Var name="us-west-2" /> ec2 describe-images --owners 126027368216 --filters 'Name=name,Values=gravitational-teleport-ami-ent-(=teleport.version=)'
+  ```
+
+  List Enterprise FIPS 140-2 AMIs<br/>
+  ```code
+  aws --region <Var name="us-west-2" /> ec2 describe-images --owners 126027368216 --filters 'Name=name,Values=gravitational-teleport-ami-ent-(=teleport.version=)-fips'
+  ```
+</Admonition>
+
+### key_name
+
+```code
+export TF_VAR_key_name="exampleuser"
+```
+
+The AWS keypair name to use when deploying EC2 instances. This must exist in the same region as you
+specify in the `region` variable, and you will need a copy of this keypair available to connect to the deployed
+EC2 instances. Do not use a keypair that you do not have access to.
+
+### license_path
+
+(!docs/pages/includes//enterprise/obtainlicense.mdx!)
+
+```code
+export TF_VAR_license_path="/home/user/license.pem"
+```
+
+This license will be uploaded to AWS SSM and automatically downloaded to Teleport auth nodes in order to enable
+Teleport Enterprise/Pro functionality.
+
+(OSS users can provide any valid local file path here - it isn't used by the auth server in a Teleport OSS install)
```

Comment:
I would leave a single no-op file path that OSS users can use here so they don't need to come up with one (e.g., `./no-op` or whatever passes validation).